numpy-stubs/__init__.pyi

```diff
@@ -387,6 +394,58 @@ class ndarray(_ArrayOrScalarCommon, Iterable, Sized, Container):
     def ravel(self, order: str=...) -> ndarray: ...
     def squeeze(self, axis: Union[int, Tuple[int, ...]]=...) -> ndarray: ...
 
+    # Item selection and manipulation
+    @overload
+    def take(self, indices: int, *, mode: str=...) -> Any: ...
+    @overload
+    def take(self, indices: int, axis: int, out: ndarray=..., mode: str=...) -> Any: ...
+    @overload
+    def take(self, indices: Sequence[int], axis: int=..., out: ndarray=..., mode: str=...) -> ndarray: ...
```

Comment:
`indices` can take array-like arguments, not just `Sequence`.

Does `np.ndarray` pass `Sequence` checks?

Reply:
Oh! Great finding, `ndarray` has all `Sequence`'s methods so I'll specify `Sequence` as a super-class